buildstream/_platform/linux.py

```diff
@@ -22,13 +22,21 @@
 
 from .. import _site
 from .. import utils
-from ..sandbox import SandboxBwrap
+from ..sandbox import SandboxBwrap, SandboxDummy
 
 from . import Platform
 
 
 class Linux(Platform):
 
+    ARCHITECTURES = {
```

Comment:
I know it was called like this before so it's not really a change but not entirely obvious from name that this is just an alias mapping and unlisted things may be supported.